invisible_cities/reco/paolina_functions.py

```diff
@@ -35,8 +35,8 @@ def voxelize_hits(hits             : Sequence[BHit],
                   voxel_dimensions : np.ndarray,
                   strict_voxel_size: bool = False) -> List[Voxel]:
     """1. Hits are enclosed by a bounding box.
-       2. Boundix box is discretized (via a hitogramdd).
-       3. The energy of all the hits insidex each discreet "voxel" is added.
+       2. Bounding box is discretized (via a histogramdd).
```

Comment:
@paolafer Thanks for fixing all the typos in this docstring.

@jjgomezcadenas The purpose of docstrings is to document the behaviour of a component, in order to enable a user to understand how the component should be used: docstrings should document the interface rather than the implementation. This docstring does exactly the opposite, it describes *how* `voxelize_hits` achieves its goals, but it doesn't describe at all what those goals are or how a user is expected to use `voxelize_hits`. This shouldn't be a docstring at all, it should be a comment.

Please understand that docstrings and comments serve different purposes, and don't confuse the two. Docsrtings are for clients, comments are for implementors.

Docstrings describe to users of magic black boxes, what they can expect the black box to do for them. The whole point of magic black boxes is that they work by magic: the user isn't supposed to understand or care about what goes on inside. This is a cornerstone of good program design. Consequently docstrings shouldn't talk about what happens in the box.